ctx/ctx.go

```diff
@@ -8,8 +8,10 @@ import (
 	"github.com/TykTechnologies/tyk/user"
 )
 
+type Key uint
```

Comment:
More descriptive name like `ContextKey`

Reply:
The package is `ctx` so it looks descriptive enough since you refer to it as `ctx.Key`

Reply:
I didn't realize it is in `ctx` package, ok.